lib/system/excpt.nim

```diff
@@ -229,16 +229,20 @@ proc auxWriteStackTrace(f: PFrame; s: var seq[StackTraceEntry]) =
   while it != nil:
     s[last] = StackTraceEntry(procname: it.procname,
                               line: it.line,
+                              col: it.col,
                               filename: it.filename)
     it = it.prev
     dec last
 
-template addFrameEntry(s, f: untyped) =
+template addFrameEntry(s: var string, f: PFrame|StackTraceEntry) =
   var oldLen = s.len
   add(s, f.filename)
   if f.line > 0:
     add(s, '(')
+    # todo: reuse `lineInfoToString`
     add(s, $f.line)
+    add(s, ", ")
+    add(s, $(f.col+1))
```

Comment:
Nitpick: The C codegen should emit the "correct" column with a ``+ 1``.

Reply:
that'd be inconsistent with `TLineInfo` which has 0-based cols (which later have to be +1'd to be printed), so I prefer to keep it as is.
A potential cleanup solution (for another PR) to avoid all those off by 1 errors (eg see my previous bugfix https://github.com/nim-lang/Nim/pull/10138) is to introduce:
`type HumanInt = distinct int` which is 1-based integer; anything else is `0-based`